platform/machine/util/util.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"context"
+
+	"github.com/coreos/mantle/platform"
+)
+
+func Reboot(m platform.Machine, j *platform.Journal) error {
+	if err := platform.StartReboot(m); err != nil {
+		return err
+	}
+	return Start(m, j)
+}
+
+func Start(m platform.Machine, j *platform.Journal) error {
+	if err := j.Start(context.TODO(), m); err != nil {
+		return err
+	}
+	if err := platform.CheckMachine(m); err != nil {
+		return err
```

Comment:
We need to carry over the `failed basic checks` message from some of the platforms, since that case occurs frequently.  It wouldn't hurt to wrap the other errs while you're here.